src/content/docs/victory-histogram.md

```diff
@@ -0,0 +1,559 @@
+---
+id: 35
+title: VictoryHistogram
+category: charts
+type: docs
+scope:
+  - sampleHistogramData
+  - sampleHistogramDateData
+  - d3Scale
+  - d3Time
+  - d3Array
+---
+
+# VictoryHistogram
+
+VictoryHistogram renders a dataset as series of bars representing "bins", allowing you to see the distribution of your data. The data you pass in will be "binned" according to the `bin` prop you pass, allowing for flexibility in how these bins are determined. VictoryHistogram can be composed with [`VictoryChart`][] to create histogram charts.
+
+Because each bar represents a bin rather than a particular data point (like you would have with `VictoryScatter` for example), when accessing `datum` via a prop that passes `datum` such as `style`, datum will have properties `x`, `x0`, `x1`, `y`, and `binnedData`. `x` is the midpoint between the bin, `x0` is the beginning of the bin, `x1` is the end of the bin, `y` is the aggregate amount of data points within that bin, and `binnedData` is an array of the original data points that were grouped into this bin.
+
+```playground
+<VictoryChart
+  theme={VictoryTheme.material}
+  domainPadding={10}
+>
+  <VictoryHistogram
+    style={{ data: { fill: "#c43a31" } }}
+    data={sampleHistogramData}
+  />
+</VictoryChart>
+```
+
+## animate
+
+`type: boolean || object`
+
+`VictoryHistogram` uses the standard `animate` prop. [Read about it here](/docs/common-props#animate)
+
+See the [Animations Guide][] for more detail on animations and transitions
+
+```jsx
+animate={{
+  duration: 2000,
+  onLoad: { duration: 1000 }
+}}
+```
+
+## bins
+
+`type: number || array[numbers] || array[dates]`
+
+_default:_ `undefined`
+
+**`VictoryHistogram` uses [`d3.bin`](https://observablehq.com/@d3/d3-bin) to do binning.**
+
+The `bins` prop is used to specify how the data will be binned. There are a few options for this, the first being passing no value, ie the default behavior, which is letting `d3.bin` generate the buckets based on the data. The second is passing a number, which specifies _approximately_ the number of bins to generate, this is not a guarantee (see `d3.bin` for more details). The last options are passing an array of numbers or dates (depending on your data), this array represents an array of thresholds. So for example if you passed `[0, 10, 20, 35]`, you'd get 3 bins, that would look like [0, 10) , [10, 20), \[20, 35\].
```

Comment:
example bins have mixed brackets and parens

Reply:
I was trying to denote inclusive/exclusive, to show how the array version of the `bin` prop is used to generate the bins.